packages/css-library/dist/stylesheets/base/va.css

```diff
@@ -1834,7 +1834,7 @@ article > h1 {
   line-height: 1.7;
   max-width: 48.125rem;
 }
-.va-introtext p {
+.va-introtext p, .va-introtext a {
```

Comment:
This is being generated with `yarn build` so I think a previous build just wasn't fully checked in in a past PR. Getting it in now.